snorkel/matchers.py

```diff
@@ -110,7 +111,10 @@ def _f(self, c):
         p = c.get_attrib_span(self.attrib)
         p = p.lower() if self.ignore_case else p
         p = self._stem(p) if self.stemmer is not None else p
-        return True if p in self.d else False
+        if self.reverse:
+            return False if p in self.d else True
+        else:
+            return True if p in self.d else False
```

Comment:
![giphy](https://cloud.githubusercontent.com/assets/6849058/17717971/20ebf816-63c6-11e6-8f36-f6196c14f666.gif)
Mind blown. Yep, that would work! Should I implement the change?